<?xml version='1.0' encoding='UTF-8'?>
<feed xmlns="http://www.w3.org/2005/Atom">
  <id>https://raw.githubusercontent.com/ajmaradiaga/feeds/main/scmt/topics/SAP-Connectivity-service-qa.xml</id>
  <title>SAP Community - SAP Connectivity service</title>
  <updated>2025-09-29T23:12:31.909921+00:00</updated>
  <link href="https://community.sap.com/t5/c-khhcw49343/SAP Connectivity service/pd-p/67837800100800004901" rel="alternate"/>
  <generator uri="https://lkiesow.github.io/python-feedgen" version="1.0.0">python-feedgen</generator>
  <subtitle>SAP Connectivity service Q&amp;A in SAP Community</subtitle>
  <entry>
    <id>https://community.sap.com/t5/technology-q-a/i-am-having-trouble-trying-to-create-a-program-in-se38-merp-core-cpms/qaq-p/13894211</id>
    <title>I am having trouble trying to create a program in SE38: /MERP/CORE_CPMS_APPCREAT_PROG.</title>
    <updated>2024-10-09T16:46:51.595000+02:00</updated>
    <author>
      <name>Luis1001</name>
      <uri>https://community.sap.com/t5/user/viewprofilepage/user-id/1534145</uri>
    </author>
    <content>&lt;P&gt;Hello, specialist,&lt;/P&gt;&lt;P&gt;I am facing a problem when trying to generate the SAP&amp;amp;SAM_2205 application in the program SE38: /MERP/CORE_CPMS_APPCREATE_PROG. It is returning a "connection refused" error.&lt;BR /&gt;&lt;BR /&gt;&lt;/P&gt;&lt;P&gt;&lt;span class="lia-inline-image-display-wrapper lia-image-align-inline" image-alt="Luis1001_0-1728484840587.png" style="width: 400px;"&gt;&lt;img src="https://community.sap.com/t5/image/serverpage/image-id/177564i83EC0B6E11CD9077/image-size/medium?v=v2&amp;amp;px=400" role="button" title="Luis1001_0-1728484840587.png" alt="Luis1001_0-1728484840587.png" /&gt;&lt;/span&gt;&lt;/P&gt;&lt;P&gt;&lt;span class="lia-inline-image-display-wrapper lia-image-align-inline" image-alt="Luis1001_1-1728484857693.png" style="width: 400px;"&gt;&lt;img src="https://community.sap.com/t5/image/serverpage/image-id/177565i1F5F6681CEDE09EE/image-size/medium?v=v2&amp;amp;px=400" role="button" title="Luis1001_1-1728484857693.png" alt="Luis1001_1-1728484857693.png" /&gt;&lt;/span&gt;&lt;/P&gt;&lt;P&gt;Upon researching, I found a note suggesting to run this program in SE38: /SYCLO/CORE_EXCH_PURGE_PROG.&lt;/P&gt;&lt;P&gt;It is returning the following errors:&lt;BR /&gt;&lt;BR /&gt;&lt;/P&gt;&lt;P&gt;&lt;span class="lia-inline-image-display-wrapper lia-image-align-inline" image-alt="Luis1001_2-1728484896653.png" style="width: 400px;"&gt;&lt;img src="https://community.sap.com/t5/image/serverpage/image-id/177566i8D6993E374959522/image-size/medium?v=v2&amp;amp;px=400" role="button" title="Luis1001_2-1728484896653.png" alt="Luis1001_2-1728484896653.png" /&gt;&lt;/span&gt;&lt;/P&gt;&lt;P&gt;Could someone help me solve this problem?&lt;/P&gt;&lt;P&gt;-Thank you&lt;/P&gt;&lt;P&gt;&lt;BR /&gt;&lt;BR /&gt;&lt;BR /&gt;&lt;/P&gt;</content>
    <link href="https://community.sap.com/t5/technology-q-a/i-am-having-trouble-trying-to-create-a-program-in-se38-merp-core-cpms/qaq-p/13894211"/>
    <published>2024-10-09T16:46:51.595000+02:00</published>
  </entry>
  <entry>
    <id>https://community.sap.com/t5/technology-q-a/connecting-to-remote-on-premise-system-locally-from-bas-through-principal/qaq-p/13914728</id>
    <title>Connecting to remote on-premise system locally from BAS through Principal Propagation</title>
    <updated>2024-10-22T00:31:11.497000+02:00</updated>
    <author>
      <name>FranciscoAcu√±a</name>
      <uri>https://community.sap.com/t5/user/viewprofilepage/user-id/173604</uri>
    </author>
    <content>&lt;P&gt;Hello CAP experts,&amp;nbsp;&lt;BR /&gt;&lt;BR /&gt;I am working on a CAP project that needs to integrate data from several different sources, one of them being an on-premise S/4HANA system which I am trying to consume through a &lt;STRONG&gt;BTP destination with Principal Propagation as its authentication type&lt;/STRONG&gt;.&lt;BR /&gt;&lt;BR /&gt;For guidance, I've been following &lt;A href="https://community.sap.com/t5/technology-blogs-by-sap/btp-cap-how-to-connect-to-remote-services-locally-in-cap-application/ba-p/13550070" target="_self"&gt;this blog&lt;/A&gt;, and &lt;A href="https://cap.cloud.sap/docs/guides/using-services?q=cds+bind#connect-to-remote-services-locally" target="_self" rel="nofollow noopener noreferrer"&gt;this entry from CAP's official documentation&lt;/A&gt;.&lt;/P&gt;&lt;P&gt;I used these commands to create cloud foundry XSUAA and Destination service instances and their respective service keys:&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;pre class="lia-code-sample language-bash"&gt;&lt;code&gt;cf create-service xsuaa application Intercompany-Metrics-auth
cf create-service-key Intercompany-Metrics-auth Intercompany-Metrics-auth-key

cf create-service destination lite Intercompany-Metrics-destination
cf create-service-key Intercompany-Metrics-destination Intercompany-Metrics-destination-key&lt;/code&gt;&lt;/pre&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;P&gt;To run the app, I use these commands to first bind to these created services and my HANA DB HDI container then go into hybrid testing mode:&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;pre class="lia-code-sample language-bash"&gt;&lt;code&gt;cds bind -2 Intercompany-Metrics-auth,Intercompany-Metrics-destination,Intercompany-Metrics-dev:SharedDevKey

cds watch --profile hybrid&lt;/code&gt;&lt;/pre&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;P&gt;&amp;nbsp;This is how my cdsrc-private.json file looks like after running the previous cds bind command.&lt;BR /&gt;&lt;BR /&gt;&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;pre class="lia-code-sample language-json"&gt;&lt;code&gt;{
  "requires": {
    "[hybrid]": {
      "auth": {
        "binding": {
          "type": "cf",
          "apiEndpoint": "https://api.cf.eu10-004.hana.ondemand.com",
          "org": "UAT",
          "space": "S4 Extensions",
          "instance": "Intercompany-Metrics-auth",
          "key": "Intercompany-Metrics-auth-key",
          "resolved": false
        },
        "kind": "xsuaa-auth",
        "vcap": {
          "name": "auth"
        }
      },
      "destinations": {
        "binding": {
          "type": "cf",
          "apiEndpoint": "https://api.cf.eu10-004.hana.ondemand.com",
          "org": "UAT",
          "space": "S4 Extensions",
          "instance": "Intercompany-Metrics-destination",
          "key": "Intercompany-Metrics-destination-key",
          "resolved": false
        },
        "kind": "destinations",
        "vcap": {
          "name": "destinations"
        }
      },
      "Z_PNL_STATEMENT_CDS": {
        "kind": "odata-v2",
        "model": "srv/external/Z_PNL_STATEMENT_CDS",
        "credentials": {
          "path": "/sap/opu/odata/sap/Z_PNL_STATEMENT_CDS",
          "destination": "s4destrt"
        }
      },
      "db": {
        "binding": {
          "type": "cf",
          "apiEndpoint": "https://api.cf.eu10-004.hana.ondemand.com",
          "org": "UAT",
          "space": "S4 Extensions",
          "instance": "Intercompany-Metrics-dev",
          "key": "SharedDevKey",
          "resolved": false
        },
        "kind": "hana-cloud",
        "vcap": {
          "name": "db"
        }
      }
    }
  }
}&lt;/code&gt;&lt;/pre&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;P&gt;When running the app, and trying to perform a simple query onto my on-premise system:&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;pre class="lia-code-sample language-javascript"&gt;&lt;code&gt;this.on('READ', LiveMetrics, async (req) =&amp;gt; {

        const PNLService = await cds.connect.to('Z_PNL_STATEMENT_CDS');
        // Fetch data from the external OData service
        const results = await PNLService.run(
            SELECT.from('Z_PNL_AREA')
        );

        return results;

    });&lt;/code&gt;&lt;/pre&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;P&gt;I receive the following Failed to load destination error:&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;pre class="lia-code-sample language-bash"&gt;&lt;code&gt;message: "Error during request to remote service: Failed to load destination. Caused by: No user token (JWT) has been provided. This is strictly necessary for 'PrincipalPropagation'.",
    request: {
      method: 'GET',
      url: '/sap/opu/odata/sap/Z_PNL_STATEMENT_CDS/Z_PNL_AREA',
      headers: [Object]
    },&lt;/code&gt;&lt;/pre&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;P data-unlink="true"&gt;&amp;nbsp;I've also tried using my project's&amp;nbsp;App Router. Without stopping my running app, I opened a second terminal and started my app router with this command hoping to bind my router to the same XSUAA and destination service I created earlier:&lt;BR /&gt;&lt;BR /&gt;&lt;/P&gt;&lt;pre class="lia-code-sample language-bash"&gt;&lt;code&gt;cds bind -2 Intercompany-Metrics-auth,Intercompany-Metrics-destination,Intercompany-Metrics-db:SharedDevKey --exec -- npm start --prefix app/router&lt;/code&gt;&lt;/pre&gt;&lt;P data-unlink="true"&gt;This returns this "Failed to load destination" error:&lt;/P&gt;&lt;pre class="lia-code-sample language-bash"&gt;&lt;code&gt;message: 'Error during request to remote service: Failed to load destination. Caused by: No binding to a connectivity service found! Please make sure to bind an instance of the connectivity service to your app if you want to connect to on-premise destinations.',&lt;/code&gt;&lt;/pre&gt;&lt;P data-unlink="true"&gt;&lt;BR /&gt;&lt;BR /&gt;&lt;BR /&gt;&lt;BR /&gt;&lt;/P&gt;&lt;P data-unlink="true"&gt;&amp;nbsp;&lt;/P&gt;</content>
    <link href="https://community.sap.com/t5/technology-q-a/connecting-to-remote-on-premise-system-locally-from-bas-through-principal/qaq-p/13914728"/>
    <published>2024-10-22T00:31:11.497000+02:00</published>
  </entry>
  <entry>
    <id>https://community.sap.com/t5/technology-q-a/communication-arrangement-in-sap/qaq-p/13932462</id>
    <title>Communication arrangement in SAP</title>
    <updated>2024-11-08T12:19:47.729000+01:00</updated>
    <author>
      <name>pranavi3</name>
      <uri>https://community.sap.com/t5/user/viewprofilepage/user-id/1803737</uri>
    </author>
    <content>&lt;P&gt;Hello Experts,&lt;/P&gt;&lt;P&gt;We are facing connection issue while creating communication arrangement in SAP,&lt;/P&gt;&lt;P&gt;issue:&amp;nbsp;&lt;U&gt;SOAP:1032 SRT: Wrong Content-Type and empty HTTP-B ody received: ("HTTP Code 401 : Unauthorized")&lt;/U&gt;&lt;/P&gt;&lt;P&gt;we tried changing user ID and password but it's still the same.&lt;/P&gt;&lt;P&gt;&amp;nbsp;Please suggest us on how to solve this error.&lt;/P&gt;&lt;P&gt;Regards&lt;/P&gt;&lt;P&gt;Pranavi Reddy&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;</content>
    <link href="https://community.sap.com/t5/technology-q-a/communication-arrangement-in-sap/qaq-p/13932462"/>
    <published>2024-11-08T12:19:47.729000+01:00</published>
  </entry>
  <entry>
    <id>https://community.sap.com/t5/technology-q-a/clientcredentials-grant-flow-with-one-xsuaa-and-multiple-apps-inlcuding/qaq-p/13935706</id>
    <title>ClientCredentials grant flow with one XSUAA and multiple apps inlcuding scopes</title>
    <updated>2024-11-12T13:50:01.462000+01:00</updated>
    <author>
      <name>kosmopilot</name>
      <uri>https://community.sap.com/t5/user/viewprofilepage/user-id/807596</uri>
    </author>
    <content>&lt;P&gt;I would like to do server-to-server communication on the BTP (CF) using the ClientCredentials grant flow with custom scopes. For example, consider a &lt;STRONG&gt;client&lt;/STRONG&gt; app and a &lt;STRONG&gt;server&lt;/STRONG&gt; app that are bound to the same XSUAA and destination service instance as they are deployed in a single manifest.yaml. During deployment, a destination is created for the server API with the ClientCredentials grant flow as authentication.&lt;/P&gt;&lt;P&gt;The error occurs when the destination is fetched from the Destination Service. Then, the ClientCredentails grant flow is executed by the Destination Service and fails as the scope "Test.Admin" is invalid. Omitting the scope lets the flow complete, but the access token does not include the scope and thus endpoints of the server app which require the "Admin" role can not be called.&lt;/P&gt;&lt;P&gt;The destination looks like this:&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;pre class="lia-code-sample language-javascript"&gt;&lt;code&gt;tokenServiceURLType=Dedicated
clientId=exampleClientId
clientSecret=exampleClientSecret
xsappname=exampleAppName
tokenServiceURL=https\://60ccf4actrial.authentication.us10.hana.ondemand.com/oauth/token
WebIDEEnabled=true
HTML5.ForwardAuthToken=false
TokenServiceURLType=Dedicated
URL=https\://exampleURL.cfapps.us10-001.hana.ondemand.com
Name=server-api
Type=HTTP
verificationkey=***
Authentication=OAuth2ClientCredentials
scope=Test.Admin
ProxyType=Internet&lt;/code&gt;&lt;/pre&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;P&gt;The xs-security.json for the single, shared XSUAA instance looks like this:&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;pre class="lia-code-sample language-json"&gt;&lt;code&gt;{
  "xsappname": "${default-xsappname}",
  "tenant-mode": "dedicated",
  "scopes": [
    {
      "name": "Test.Admin"
    }
  ],
  "attributes": [],
  "role-templates": [
    {
      "name": "Admin",
      "description": "",
      "scope-references": ["Test.Admin"],
      "attribute-references": []
    }
  ],
  "oauth2-configuration": {
    "redirect-uris": ["https://*.hana.ondemand.com/**"]
  },
  "authorities": ["$ACCEPT_GRANTED_AUTHORITIES", "Test.Admin"]
}&lt;/code&gt;&lt;/pre&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;P&gt;&amp;nbsp;&amp;nbsp;&lt;/P&gt;&lt;P&gt;The issue can be manually reproduced, simulating what the Destination Service does when fetching the destination:&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;pre class="lia-code-sample language-javascript"&gt;&lt;code&gt;// Fetch token from XSUAA using ClientCredentials grant flow
GET {{tokenServiceURL}}?grant_type=client_credentials&amp;amp;scope=Test.Admin
Authorization: Basic exampleClientId:exampleClientSecret&lt;/code&gt;&lt;/pre&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;P&gt;Additionally, the simplified manifest.yaml looks like this:&lt;/P&gt;&lt;pre class="lia-code-sample language-yaml"&gt;&lt;code&gt;_schema-version: "3.1"
ID: example
version: 1.0.0
description: "Example project for server-to-server communication in background jobs"
parameters:
  enable-parallel-deployments: true

modules:
  - name: client
    # ....
    requires:
      - name: example-uaa
      - name: example-destination
      - name: server
    provides:
      - name: client-api
        properties:
          srv-url: ${default-url}

  - name: server
    # ...
    requires:
      - name: example-uaa
      - name: example-destination
    provides:
      - name: server-api
        properties:
          srv-url: ${default-url}

  # ---------------------- DESTINATIONS --------------------------
  - name: example-destination-content
    # ------------------------------------------------------------
    type: com.sap.application.content
    build-parameters:
      no-source: true
    requires:
      - name: example-destination
        parameters:
          content-target: true
      - name: server-api
      - name: example-uaa
        parameters:
          service-key:
            name: example-uaa-key
    parameters:
      content:
        instance:
          existing_destinations_policy: update
          destinations:
            - Name: server-api
              URL: ~{server-api/srv-url}
              WebIDEEnabled: true
              Authentication: OAuth2ClientCredentials
              HTML5.ForwardAuthToken: false # when enabled, the ClientCredentials flow is not triggered
              TokenServiceURLType: Dedicated
              TokenServiceInstanceName: example-uaa
              TokenServiceKeyName: example-uaa-key

resources:
  - name: example-uaa
    type: org.cloudfoundry.managed-service
    parameters:
      service-name: example-uaa
      service: xsuaa
      service-plan: application
      path: ./xs-security.json
      config:
        xsappname: example
      service-keys:
        - name: example-uaa-key
  - name: example-destination
    type: org.cloudfoundry.managed-service
    parameters:
      service-name: example-destination
      service: destination
      service-plan: lite&lt;/code&gt;&lt;/pre&gt;&lt;P&gt;Is this even possible with a single shared XSUAA instance or do I need to create one XSUAA instance for each app?&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;</content>
    <link href="https://community.sap.com/t5/technology-q-a/clientcredentials-grant-flow-with-one-xsuaa-and-multiple-apps-inlcuding/qaq-p/13935706"/>
    <published>2024-11-12T13:50:01.462000+01:00</published>
  </entry>
  <entry>
    <id>https://community.sap.com/t5/spend-management-q-a/sap-integration-suite-test-connectivity-to-sap-ariba-how-to-get-host-url/qaq-p/13936227</id>
    <title>SAP Integration Suite Test Connectivity to SAP Ariba - How to get Host URL</title>
    <updated>2024-11-12T23:50:30.108000+01:00</updated>
    <author>
      <name>TheAnji</name>
      <uri>https://community.sap.com/t5/user/viewprofilepage/user-id/1673328</uri>
    </author>
    <content>&lt;P&gt;Hello Experts ..&lt;/P&gt;&lt;P&gt;We tried to Integrate SAP Ariba to ServiceNow using SAP BTP Integration Suite.&lt;/P&gt;&lt;P&gt;As a first step, trying to connect SAP BTP Integration Suite to SAP Ariba thru "Connectivity Tests".&lt;/P&gt;&lt;P&gt;Question: How do we get Host url of SAP Ariba?&lt;/P&gt;&lt;P&gt;We tried with &lt;A href="https://S1.Ariba.com" target="_blank" rel="nofollow noopener noreferrer"&gt;https://S1.Ariba.com&lt;/A&gt;&amp;nbsp;etc.. but getting "Couldn't connect to Hot".&lt;/P&gt;&lt;P&gt;Could you please help.&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;</content>
    <link href="https://community.sap.com/t5/spend-management-q-a/sap-integration-suite-test-connectivity-to-sap-ariba-how-to-get-host-url/qaq-p/13936227"/>
    <published>2024-11-12T23:50:30.108000+01:00</published>
  </entry>
  <entry>
    <id>https://community.sap.com/t5/technology-q-a/sap-capm-service-failing-with-http-407-proxy-authentication-failed-error/qaq-p/13948589</id>
    <title>SAP CAPM service failing with http 407 Proxy Authentication failed Error</title>
    <updated>2024-11-26T11:05:09.238000+01:00</updated>
    <author>
      <name>athulramakrishnan</name>
      <uri>https://community.sap.com/t5/user/viewprofilepage/user-id/855035</uri>
    </author>
    <content>&lt;P&gt;I am consuming a SOAP based web service from a S/4HANA On-prem system inside a SAP CAP service using &lt;SPAN&gt;sap-cloud-sdk api's coupled with&amp;nbsp;&lt;/SPAN&gt;BTP destinations and cloud connector.&amp;nbsp;&lt;/P&gt;&lt;P&gt;Reference :&lt;A href="https://community.sap.com/t5/technology-blogs-by-sap/consuming-soap-web-services-in-sap-cloud-application-programming-model-cap/ba-p/13522820" target="_blank"&gt;https://community.sap.com/t5/technology-blogs-by-sap/consuming-soap-web-services-in-sap-cloud-application-programming-model-cap/ba-p/13522820&lt;/A&gt;&lt;/P&gt;&lt;P&gt;I have recently upgraded the&amp;nbsp;&lt;SPAN&gt;&lt;a href="https://community.sap.com/t5/user/viewprofilepage/user-id/1862993"&gt;@SAP&lt;/a&gt;-cloud-sdk/connectivity version to 3.23.0(latest) recently and now the service is failing sometimes with http status code 407-&amp;nbsp;Proxy Authentication failed. The service works most of the time but randomly fails with this error. And once i redeploy the CAP app to CF again the service starts working like usual.&lt;/SPAN&gt;&lt;/P&gt;</content>
    <link href="https://community.sap.com/t5/technology-q-a/sap-capm-service-failing-with-http-407-proxy-authentication-failed-error/qaq-p/13948589"/>
    <published>2024-11-26T11:05:09.238000+01:00</published>
  </entry>
  <entry>
    <id>https://community.sap.com/t5/technology-q-a/run-nodejs-app-locally-using-destination-and-connectivity-services/qaq-p/13988291</id>
    <title>Run NodeJS app locally using destination and connectivity services</title>
    <updated>2025-01-16T18:20:33.193000+01:00</updated>
    <author>
      <name>simonblaser_haufe</name>
      <uri>https://community.sap.com/t5/user/viewprofilepage/user-id/890635</uri>
    </author>
    <content>&lt;P&gt;Hello everyone,&lt;/P&gt;&lt;P&gt;lately I have been trying to create a NodeJS app using NestJS and the SAP Cloud SDK for javascript and deploy it to the cloud foundry runtime according to this example:&amp;nbsp;&lt;A href="https://github.com/SAP-samples/cloud-sdk-js?tab=readme-ov-file" target="_blank" rel="nofollow noopener noreferrer"&gt;https://github.com/SAP-samples/cloud-sdk-js?tab=readme-ov-file&lt;/A&gt;&lt;/P&gt;&lt;P&gt;I created a new service, generated the odata client using an edmx file of our on premise service and deployed it.&amp;nbsp;&lt;/P&gt;&lt;P&gt;The deployed version is fine: I can read data and consume the odata service.&amp;nbsp;&lt;BR /&gt;In local mode the destination fails.&amp;nbsp;&lt;/P&gt;&lt;P&gt;I found this blogpost telling me how to check the connectivity of my destination.&amp;nbsp;&lt;BR /&gt;Using the curl request results in this:&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;pre class="lia-code-sample language-bash"&gt;&lt;code&gt;curl -v -i "&amp;lt;destinationName&amp;gt;.dest/sap/opu/odata/iwfnd/catalogservice;v=2/ServiceCollection?%24top=1"
* Uses proxy env variable no_proxy == 'localhost,127.0.0.1,github.com,.github.com,.npmjs.org,.yarnpkg.com,npm.sap.com,.maven.apache.org,.repo-cache.svc.cluster.local'
* Uses proxy env variable http_proxy == 'http://127.0.0.1:8887'
*   Trying 127.0.0.1:8887...
* Connected to 127.0.0.1 (127.0.0.1) port 8887 (#0)
&amp;gt; GET http://&amp;lt;destinationName&amp;gt;.dest/sap/opu/odata/iwfnd/catalogservice;v=2/ServiceCollection?%24top=1 HTTP/1.1
&amp;gt; Host: sd_orca_sap_ecc_akab.dest
&amp;gt; User-Agent: curl/7.88.1
&amp;gt; Accept: */*
&amp;gt; Proxy-Connection: Keep-Alive
&amp;gt; 
&amp;lt; HTTP/1.1 500 Internal Server Error
HTTP/1.1 500 Internal Server Error
&amp;lt; Content-Type: text/plain; charset=utf-8
Content-Type: text/plain; charset=utf-8
&amp;lt; X-Content-Type-Options: nosniff
X-Content-Type-Options: nosniff
&amp;lt; Date: Thu, 16 Jan 2025 17:10:28 GMT
Date: Thu, 16 Jan 2025 17:10:28 GMT
&amp;lt; Content-Length: 75
Content-Length: 75

&amp;lt; 
dial tcp: lookup &amp;lt;destinationName&amp;gt;.dest on 100.64.0.10:53: no such host
* Connection #0 to host 127.0.0.1 left intact&lt;/code&gt;&lt;/pre&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;P&gt;I also created a .env file that should change the proxy settings:&lt;/P&gt;&lt;pre class="lia-code-sample language-abap"&gt;&lt;code&gt;destinations="[{"name": "&amp;lt;destinationName&amp;gt;","url": "http://&amp;lt;destinationName&amp;gt;.dest","proxyConfiguration":{"host": "127.0.0.1","port": 8887,"protocol":"http"}}]"&lt;/code&gt;&lt;/pre&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;P&gt;My destination looks like this:&lt;/P&gt;&lt;P&gt;&lt;span class="lia-inline-image-display-wrapper lia-image-align-inline" image-alt="simonblaser_haufe_0-1737047642238.png" style="width: 400px;"&gt;&lt;img src="https://community.sap.com/t5/image/serverpage/image-id/213162iEF5A321F05A11B68/image-size/medium?v=v2&amp;amp;px=400" role="button" title="simonblaser_haufe_0-1737047642238.png" alt="simonblaser_haufe_0-1737047642238.png" /&gt;&lt;/span&gt;&lt;/P&gt;&lt;P&gt;The destination is created on subaccount level.&amp;nbsp;&lt;BR /&gt;After changing the destination i also called the reload request Andre mentioned in his blogpost.&lt;/P&gt;&lt;P&gt;My default-env.json and default-services.json files both contain the configuration to the destination, connectivity and xsuaa service I created for the app. The variables are created using the cf env myapp &amp;gt; .env command.&amp;nbsp;&lt;BR /&gt;I am quite sure the service is found. Removing the destination service from these files results in another error: "Could not find service binding of type 'destination'".&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;P&gt;At this point I am very desperate in trying to start my app locally.&amp;nbsp;&lt;/P&gt;&lt;P&gt;Any help is appreciated, thank you already in advance!&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;</content>
    <link href="https://community.sap.com/t5/technology-q-a/run-nodejs-app-locally-using-destination-and-connectivity-services/qaq-p/13988291"/>
    <published>2025-01-16T18:20:33.193000+01:00</published>
  </entry>
  <entry>
    <id>https://community.sap.com/t5/technology-q-a/communicate-from-sap-public-cloud-to-java-application-jco-server-via/qaq-p/13999305</id>
    <title>Communicate from SAP Public Cloud to Java Application (Jco Server) via WebSocketRFC Problem</title>
    <updated>2025-01-28T14:28:12.902000+01:00</updated>
    <author>
      <name>andrzej_rosinski</name>
      <uri>https://community.sap.com/t5/user/viewprofilepage/user-id/482644</uri>
    </author>
    <content>&lt;P&gt;Hello everybody,&lt;/P&gt;&lt;P&gt;i am having a problem while trying to call a outbound service from the SAP Public Cloud to a external JCo WebSocketRFC Server instance.&amp;nbsp;&lt;/P&gt;&lt;P&gt;My JCo WebSocketRFC server is running on a VM that is exposed to the internet on port 8080.&lt;/P&gt;&lt;P&gt;In the SAP Public Cloud i have already configured the communication system where i have just entered the "Host Name" of my VM and set the Authentification Method in the "Users for Outbound Communication" to None since i do not need the authentification for the tests.&lt;/P&gt;&lt;P&gt;&lt;span class="lia-inline-image-display-wrapper lia-image-align-inline" image-alt="andrzej_rosinski_0-1738069096899.png" style="width: 626px;"&gt;&lt;img src="https://community.sap.com/t5/image/serverpage/image-id/218280i5CA10B2A390FB2B4/image-dimensions/626x80?v=v2" width="626" height="80" role="button" title="andrzej_rosinski_0-1738069096899.png" alt="andrzej_rosinski_0-1738069096899.png" /&gt;&lt;/span&gt;&lt;/P&gt;&lt;P&gt;I have also configured a communication agreement for the communication system&lt;/P&gt;&lt;P&gt;&lt;span class="lia-inline-image-display-wrapper lia-image-align-inline" image-alt="andrzej_rosinski_1-1738069224536.png" style="width: 628px;"&gt;&lt;img src="https://community.sap.com/t5/image/serverpage/image-id/218283iCBAFBAB292576C43/image-dimensions/628x142?v=v2" width="628" height="142" role="button" title="andrzej_rosinski_1-1738069224536.png" alt="andrzej_rosinski_1-1738069224536.png" /&gt;&lt;/span&gt;&lt;/P&gt;&lt;P&gt;On the Eclipse ADT side i have a communication scenario that looks like this:&lt;/P&gt;&lt;P&gt;&lt;span class="lia-inline-image-display-wrapper lia-image-align-inline" image-alt="andrzej_rosinski_2-1738069368158.png" style="width: 677px;"&gt;&lt;img src="https://community.sap.com/t5/image/serverpage/image-id/218284i85FF3530A1A0DB40/image-dimensions/677x386?v=v2" width="677" height="386" role="button" title="andrzej_rosinski_2-1738069368158.png" alt="andrzej_rosinski_2-1738069368158.png" /&gt;&lt;/span&gt;&lt;/P&gt;&lt;P&gt;with the corresponding outbound service&lt;/P&gt;&lt;P&gt;&lt;span class="lia-inline-image-display-wrapper lia-image-align-inline" image-alt="andrzej_rosinski_3-1738069410114.png" style="width: 686px;"&gt;&lt;img src="https://community.sap.com/t5/image/serverpage/image-id/218287i483C2847792264F4/image-dimensions/686x160?v=v2" width="686" height="160" role="button" title="andrzej_rosinski_3-1738069410114.png" alt="andrzej_rosinski_3-1738069410114.png" /&gt;&lt;/span&gt;&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;P&gt;Now, when i try to call the outbound service from SAP Public Cloud ABAP with the corresponding code:&lt;/P&gt;&lt;P&gt;&lt;span class="lia-inline-image-display-wrapper lia-image-align-inline" image-alt="andrzej_rosinski_6-1738070185661.png" style="width: 701px;"&gt;&lt;img src="https://community.sap.com/t5/image/serverpage/image-id/218292iC3D3040168E6C3B3/image-dimensions/701x640?v=v2" width="701" height="640" role="button" title="andrzej_rosinski_6-1738070185661.png" alt="andrzej_rosinski_6-1738070185661.png" /&gt;&lt;/span&gt;&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;P&gt;&lt;span class="lia-inline-image-display-wrapper lia-image-align-inline" image-alt="andrzej_rosinski_5-1738069869362.png" style="width: 703px;"&gt;&lt;img src="https://community.sap.com/t5/image/serverpage/image-id/218291i21E0DBE2160123FA/image-dimensions/703x559?v=v2" width="703" height="559" role="button" title="andrzej_rosinski_5-1738069869362.png" alt="andrzej_rosinski_5-1738069869362.png" /&gt;&lt;/span&gt;&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;P&gt;like described by SAP i get the following error&lt;/P&gt;&lt;P&gt;&lt;span class="lia-inline-image-display-wrapper lia-image-align-inline" image-alt="andrzej_rosinski_7-1738070655154.png" style="width: 784px;"&gt;&lt;img src="https://community.sap.com/t5/image/serverpage/image-id/218295i5A74B7666592C5B6/image-dimensions/784x16?v=v2" width="784" height="16" role="button" title="andrzej_rosinski_7-1738070655154.png" alt="andrzej_rosinski_7-1738070655154.png" /&gt;&lt;/span&gt;&lt;/P&gt;&lt;P&gt;Anyone has an idea why this fails? Dis anyone made a scenario like this before? - to communicate from the SAP Public Cloud to a external server program? I have only found tutorials how to communicate with on prem systems.&lt;/P&gt;&lt;P&gt;The JcoTrace for the server says the following (i have replaced the real IP with "hostname" in the screenshot)&lt;/P&gt;&lt;P&gt;&lt;span class="lia-inline-image-display-wrapper lia-image-align-inline" image-alt="andrzej_rosinski_1-1738137908962.png" style="width: 806px;"&gt;&lt;img src="https://community.sap.com/t5/image/serverpage/image-id/218578iECED119DD3D44DE4/image-dimensions/806x268?v=v2" width="806" height="268" role="button" title="andrzej_rosinski_1-1738137908962.png" alt="andrzej_rosinski_1-1738137908962.png" /&gt;&lt;/span&gt;&lt;/P&gt;&lt;P&gt;&lt;span class="lia-inline-image-display-wrapper lia-image-align-inline" image-alt="andrzej_rosinski_2-1738138048772.png" style="width: 800px;"&gt;&lt;img src="https://community.sap.com/t5/image/serverpage/image-id/218581iD615EE4DBCE9AF55/image-dimensions/800x266?v=v2" width="800" height="266" role="button" title="andrzej_rosinski_2-1738138048772.png" alt="andrzej_rosinski_2-1738138048772.png" /&gt;&lt;/span&gt;&lt;/P&gt;&lt;P&gt;&lt;span class="lia-inline-image-display-wrapper lia-image-align-inline" image-alt="andrzej_rosinski_4-1738138325596.png" style="width: 822px;"&gt;&lt;img src="https://community.sap.com/t5/image/serverpage/image-id/218590i061BC5E6C0183B63/image-dimensions/822x331?v=v2" width="822" height="331" role="button" title="andrzej_rosinski_4-1738138325596.png" alt="andrzej_rosinski_4-1738138325596.png" /&gt;&lt;/span&gt;&lt;/P&gt;&lt;P&gt;&lt;span class="lia-inline-image-display-wrapper lia-image-align-inline" image-alt="andrzej_rosinski_5-1738138383475.png" style="width: 837px;"&gt;&lt;img src="https://community.sap.com/t5/image/serverpage/image-id/218591i7217D96652A86118/image-dimensions/837x191?v=v2" width="837" height="191" role="button" title="andrzej_rosinski_5-1738138383475.png" alt="andrzej_rosinski_5-1738138383475.png" /&gt;&lt;/span&gt;&lt;/P&gt;&lt;P&gt;obviously the handshake cannot be completed. I am trying to connect to the server withot any authentification (i have set the communication scenario to "Unauthentifiacted" and i have startet the Jco server without any p12 files - jco.server.use_tls=0).&lt;/P&gt;&lt;P&gt;Server props:&lt;/P&gt;&lt;P&gt;&lt;span class="lia-inline-image-display-wrapper lia-image-align-inline" image-alt="andrzej_rosinski_6-1738138858738.png" style="width: 796px;"&gt;&lt;img src="https://community.sap.com/t5/image/serverpage/image-id/218597i6ABB16E876BD22FB/image-dimensions/796x217?v=v2" width="796" height="217" role="button" title="andrzej_rosinski_6-1738138858738.png" alt="andrzej_rosinski_6-1738138858738.png" /&gt;&lt;/span&gt;&lt;/P&gt;&lt;P&gt;&lt;a href="https://community.sap.com/t5/user/viewprofilepage/user-id/121864"&gt;@MarkusTolksdorf&lt;/a&gt;&amp;nbsp;&lt;a href="https://community.sap.com/t5/user/viewprofilepage/user-id/771659"&gt;@simon_luser&lt;/a&gt;&amp;nbsp;Thanks for your reply, i have added the Trace from the Jco. Could the problem be the missing authorisation?&amp;nbsp;&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;</content>
    <link href="https://community.sap.com/t5/technology-q-a/communicate-from-sap-public-cloud-to-java-application-jco-server-via/qaq-p/13999305"/>
    <published>2025-01-28T14:28:12.902000+01:00</published>
  </entry>
  <entry>
    <id>https://community.sap.com/t5/technology-q-a/sap-dbtech-jdbc-403-internal-error-cannot-get-remote-source-objects/qaq-p/14016010</id>
    <title>SAP DBTech JDBC: [403]: internal error: Cannot get remote source objects:</title>
    <updated>2025-02-13T11:07:42.145000+01:00</updated>
    <author>
      <name>Shuklaanchal</name>
      <uri>https://community.sap.com/t5/user/viewprofilepage/user-id/1929344</uri>
    </author>
    <content>&lt;P&gt;Hello Experts!&lt;/P&gt;&lt;P&gt;While connecting the remote source system, S4_HANA_SDA connect timeout expired error occur.&amp;nbsp;&lt;/P&gt;&lt;P&gt;Here is the error details.&lt;/P&gt;&lt;P&gt;&lt;span class="lia-inline-image-display-wrapper lia-image-align-center" image-alt="Screenshot 2025-02-13 153039.png" style="width: 999px;"&gt;&lt;img src="https://community.sap.com/t5/image/serverpage/image-id/225528i31F41C721DE28B11/image-size/large?v=v2&amp;amp;px=999" role="button" title="Screenshot 2025-02-13 153039.png" alt="Screenshot 2025-02-13 153039.png" /&gt;&lt;/span&gt;&lt;/P&gt;&lt;P&gt;&amp;nbsp;Thanks in advance&lt;/P&gt;</content>
    <link href="https://community.sap.com/t5/technology-q-a/sap-dbtech-jdbc-403-internal-error-cannot-get-remote-source-objects/qaq-p/14016010"/>
    <published>2025-02-13T11:07:42.145000+01:00</published>
  </entry>
  <entry>
    <id>https://community.sap.com/t5/technology-q-a/outbound-communication-from-sap-public-cloud-to-a-websocket-rfc-server-in-a/qaq-p/14034359</id>
    <title>Outbound communication from SAP Public cloud to a Websocket RFC Server in a fenced customer network</title>
    <updated>2025-03-05T12:14:51.564000+01:00</updated>
    <author>
      <name>andrzej_rosinski</name>
      <uri>https://community.sap.com/t5/user/viewprofilepage/user-id/482644</uri>
    </author>
    <content>&lt;P&gt;Hello everybody,&lt;/P&gt;&lt;P&gt;is there any possibility to call a outbound service from SAP Public Cloud to a Websocket RFC server that is&amp;nbsp;running in a fenced company network ? I have already called that Websocket RFC server with success but in that scenario the server must be accessible from the internet.&amp;nbsp;&lt;/P&gt;&lt;P&gt;Maybe there is any way to use a VPN tunnel to get that connection done? Maybe even not with Websocket RFC if that should not be possible but with HTTP or even thru BTP like:&lt;/P&gt;&lt;P&gt;SAP Public Cloud -&amp;gt; BTP (some destination or tunnel) -&amp;gt; external program in a fenced company network (server).&lt;/P&gt;&lt;P&gt;Can such scenario be accomplished at all or is it not possible?&lt;/P&gt;&lt;P&gt;Greetings!&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;P&gt;&lt;a href="https://community.sap.com/t5/user/viewprofilepage/user-id/771659"&gt;@simon_luser&lt;/a&gt;&amp;nbsp;, &lt;a href="https://community.sap.com/t5/user/viewprofilepage/user-id/15899"&gt;@Ulrich_Schmidt1&lt;/a&gt;&amp;nbsp; thanks for your replies - update 14.03.25 - in the meanwhile i have installed the Cloud Connector on the machine where my WebSocket RFC Server is running (inside a fenced company network). Sadly i still wasn't able to establish a connection. For now i am not sure if this can be done using the Cloud Connector combined with a WebSocket RFC call from SAP public cloud. The problem is, as i suppose, that in the cloud connector there is no option for a RFC protocoll while selecting the target system "Non-SAP" that is my WebSocket RFC server. the possible option are:&lt;/P&gt;&lt;P&gt;&lt;span class="lia-inline-image-display-wrapper lia-image-align-inline" image-alt="andrzej_rosinski_2-1741948504505.png" style="width: 400px;"&gt;&lt;img src="https://community.sap.com/t5/image/serverpage/image-id/237510iD5DFE95BD81BFC03/image-size/medium?v=v2&amp;amp;px=400" role="button" title="andrzej_rosinski_2-1741948504505.png" alt="andrzej_rosinski_2-1741948504505.png" /&gt;&lt;/span&gt;&lt;/P&gt;&lt;P&gt;Therefore i have selected the TCP protocol that at least does "see" the WebSoclet RFC server running on its port&lt;/P&gt;&lt;P&gt;&lt;span class="lia-inline-image-display-wrapper lia-image-align-inline" image-alt="andrzej_rosinski_1-1741948431063.png" style="width: 770px;"&gt;&lt;img src="https://community.sap.com/t5/image/serverpage/image-id/237509i270508CC4362D490/image-dimensions/770x135?v=v2" width="770" height="135" role="button" title="andrzej_rosinski_1-1741948431063.png" alt="andrzej_rosinski_1-1741948431063.png" /&gt;&lt;/span&gt;&lt;/P&gt;&lt;P&gt;but with that configuration i get no connection to the server. In the monitor view i do not see any throughput for the configured Cloud Connector.&lt;/P&gt;&lt;P&gt;I have also found a blog entry from Thomas Weiss (from 2021 - maybe its outdated?)&lt;/P&gt;&lt;P&gt;&lt;A title="WebSocket RFC blog" href="https://community.sap.com/t5/technology-blogs-by-sap/websocket-rfc-rfc-for-the-internet/ba-p/13502531" target="_self"&gt;https://community.sap.com/t5/technology-blogs-by-sap/websocket-rfc-rfc-for-the-internet/ba-p/13502531&lt;/A&gt;&amp;nbsp;&lt;/P&gt;&lt;P&gt;that is saying the following:&lt;/P&gt;&lt;P&gt;&lt;span class="lia-inline-image-display-wrapper lia-image-align-inline" image-alt="andrzej_rosinski_3-1741948684851.png" style="width: 609px;"&gt;&lt;img src="https://community.sap.com/t5/image/serverpage/image-id/237512iCBBA03FC81AD1B2D/image-dimensions/609x571?v=v2" width="609" height="571" role="button" title="andrzej_rosinski_3-1741948684851.png" alt="andrzej_rosinski_3-1741948684851.png" /&gt;&lt;/span&gt;&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;P&gt;So i suppose for the scenario:&lt;/P&gt;&lt;P&gt;--&amp;gt; SAP Public Cloud (embedded steampunk) to WebSocket RFC Server in a fenced company network some different communication type must be used.&lt;/P&gt;&lt;P&gt;Or maybe i am doing something wrong? any ideas?&lt;/P&gt;&lt;P&gt;Greetings!&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;</content>
    <link href="https://community.sap.com/t5/technology-q-a/outbound-communication-from-sap-public-cloud-to-a-websocket-rfc-server-in-a/qaq-p/14034359"/>
    <published>2025-03-05T12:14:51.564000+01:00</published>
  </entry>
  <entry>
    <id>https://community.sap.com/t5/technology-q-a/call-action-project-error-querya-enotfound-in-build-process-automation/qaq-p/14048766</id>
    <title>Call Action Project error queryA ENOTFOUND in Build Process Automation</title>
    <updated>2025-03-18T18:48:02.785000+01:00</updated>
    <author>
      <name>sakthidevi1</name>
      <uri>https://community.sap.com/t5/user/viewprofilepage/user-id/1478437</uri>
    </author>
    <content>&lt;P&gt;Hello Experts,&lt;/P&gt;&lt;P&gt;&lt;STRONG&gt;Destination configuration&amp;nbsp;&lt;/STRONG&gt;&lt;/P&gt;&lt;P&gt;we configured the S4HANA On-premise URL as a Destination like in a below screenshot.&lt;/P&gt;&lt;P&gt;&lt;STRONG&gt;&lt;span class="lia-inline-image-display-wrapper lia-image-align-inline" image-alt="sakthidevi1_0-1742318079644.png" style="width: 400px;"&gt;&lt;img src="https://community.sap.com/t5/image/serverpage/image-id/239156i9B1A676785786734/image-size/medium?v=v2&amp;amp;px=400" role="button" title="sakthidevi1_0-1742318079644.png" alt="sakthidevi1_0-1742318079644.png" /&gt;&lt;/span&gt;&lt;/STRONG&gt;&lt;/P&gt;&lt;P&gt;&lt;STRONG&gt;Action Project&lt;/STRONG&gt;&lt;/P&gt;&lt;P&gt;&lt;STRONG&gt;&lt;span class="lia-inline-image-display-wrapper lia-image-align-inline" image-alt="sakthidevi1_1-1742318224262.png" style="width: 400px;"&gt;&lt;img src="https://community.sap.com/t5/image/serverpage/image-id/239157i8E2CBBC8CE291F9D/image-size/medium?v=v2&amp;amp;px=400" role="button" title="sakthidevi1_1-1742318224262.png" alt="sakthidevi1_1-1742318224262.png" /&gt;&lt;/span&gt;&lt;/STRONG&gt;&lt;/P&gt;&lt;P&gt;&lt;STRONG&gt;Process Automation Flow&lt;/STRONG&gt;&lt;/P&gt;&lt;P&gt;We added the above Action project as a Dependency to our process automation flow like in a below screenshot.&lt;/P&gt;&lt;P&gt;&lt;STRONG&gt;&lt;span class="lia-inline-image-display-wrapper lia-image-align-inline" image-alt="sakthidevi1_2-1742318294521.png" style="width: 400px;"&gt;&lt;img src="https://community.sap.com/t5/image/serverpage/image-id/239158i94DBCC974F3C9E26/image-size/medium?v=v2&amp;amp;px=400" role="button" title="sakthidevi1_2-1742318294521.png" alt="sakthidevi1_2-1742318294521.png" /&gt;&lt;/span&gt;&lt;/STRONG&gt;&lt;/P&gt;&lt;P&gt;&lt;STRONG&gt;Problem we are facing&lt;/STRONG&gt;&lt;/P&gt;&lt;P&gt;Below is the response of the action activity 'GET_MAIL_LIST'.&lt;/P&gt;&lt;P&gt;&lt;span class="lia-inline-image-display-wrapper lia-image-align-inline" image-alt="sakthidevi1_3-1742318636789.png" style="width: 400px;"&gt;&lt;img src="https://community.sap.com/t5/image/serverpage/image-id/239163iF6EE82DA2CA0D73E/image-size/medium?v=v2&amp;amp;px=400" role="button" title="sakthidevi1_3-1742318636789.png" alt="sakthidevi1_3-1742318636789.png" /&gt;&lt;/span&gt;&lt;/P&gt;&lt;P&gt;I saw the similar issue in the below community link :&amp;nbsp;&lt;A href="Re: Call web service error queryA ENOTFOUND - SAP Community" target="_self" rel="nofollow noopener noreferrer"&gt;Call web service error queryA ENOTFOUND - SAP Community&lt;/A&gt;&amp;nbsp;However it didn't help us to resolve the issue.&lt;/P&gt;&lt;P&gt;Kindly help me to figure out the issue.&lt;/P&gt;&lt;P&gt;&lt;a href="https://community.sap.com/t5/c-khhcw49343/SAP+Business+Technology+Platform/pd-p/73555000100700000172" class="lia-product-mention" data-product="1215-1"&gt;SAP Business Technology Platform&lt;/a&gt;,&amp;nbsp;&lt;a href="https://community.sap.com/t5/c-khhcw49343/SAP+Build+Process+Automation/pd-p/73554900100800003832" class="lia-product-mention" data-product="1213-1"&gt;SAP Build Process Automation&lt;/a&gt;&amp;nbsp;&lt;/P&gt;&lt;P&gt;Thanks in advance.&lt;/P&gt;&lt;P&gt;Best Regards,&lt;/P&gt;&lt;P&gt;Sakthi.&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;</content>
    <link href="https://community.sap.com/t5/technology-q-a/call-action-project-error-querya-enotfound-in-build-process-automation/qaq-p/14048766"/>
    <published>2025-03-18T18:48:02.785000+01:00</published>
  </entry>
  <entry>
    <id>https://community.sap.com/t5/technology-q-a/error-creating-content-provider-for-sap-build-work-zone-in-consumer/qaq-p/14072765</id>
    <title>Error Creating Content Provider for SAP Build Work Zone in Consumer Subaccount</title>
    <updated>2025-04-10T18:24:21.914000+02:00</updated>
    <author>
      <name>guilherme_sales</name>
      <uri>https://community.sap.com/t5/user/viewprofilepage/user-id/17054</uri>
    </author>
    <content>&lt;P&gt;I am trying to create a multi-tenant application with SAPUI5 interface that is consumed by the subscriber's &lt;a href="https://community.sap.com/t5/c-khhcw49343/SAP+Build+Work+Zone%25252C+standard+edition/pd-p/73554900100800003081" class="lia-product-mention" data-product="684-1"&gt;SAP Build Work Zone, standard edition&lt;/a&gt;&amp;nbsp;subscription, but I am getting errors trying to create the Content Provider.&amp;nbsp;&lt;/P&gt;&lt;P&gt;Following the tutorial in the documentation&amp;nbsp;&lt;A href="https://help.sap.com/docs/build-work-zone-standard-edition/sap-build-work-zone-standard-edition/developing-html5-apps-for-cross-subaccount-consumption" target="_blank" rel="noopener noreferrer"&gt;Developing HTML5 Business Solutions as Content Providers&lt;/A&gt;, which has steps for&amp;nbsp;&lt;SPAN&gt;Multitenant consumption via subscription. I updated the "sap.cloud.service" in the SAPUI5 manifest.json file, added the cdm.json file and the script on the mta.yaml file to copy it to the resources folder. I also defined the destination templates in the provider side.&lt;/SPAN&gt;&lt;/P&gt;&lt;P&gt;&lt;SPAN&gt;After that I followed the Administrator steps in&amp;nbsp;&lt;A href="https://help.sap.com/docs/build-work-zone-standard-edition/sap-build-work-zone-standard-edition/content-providers-html5-apps-across-subaccounts" target="_self" rel="noopener noreferrer"&gt;Federation of HTML5 Business Solutions&lt;/A&gt;&amp;nbsp;documentation. Subscribed to the multi-tenant app from another subaccount in the same environment and region. Exported and imported the destinations to the consumer subaccount, changing the runtime destination URL to the subscription URL of the consumer.&lt;/SPAN&gt;&lt;/P&gt;&lt;P&gt;&lt;SPAN&gt;However after I create the Content Provider in the consumer SAP Build Work Zone channel manager, the content fetch fails with the following message:&lt;/SPAN&gt;&lt;/P&gt;&lt;P&gt;&lt;SPAN&gt;"&amp;lt;Connectivity/InternalError&amp;gt;: Could not fetch content due to an internal error. Please try again later. If the issue persists, please open a ticket on the connectivity service."&lt;BR /&gt;&lt;/SPAN&gt;&lt;/P&gt;&lt;P&gt;&lt;span class="lia-inline-image-display-wrapper lia-image-align-inline" image-alt="guilherme_sales_0-1744302150241.png" style="width: 400px;"&gt;&lt;img src="https://community.sap.com/t5/image/serverpage/image-id/249455i563CD963C51CE507/image-size/medium?v=v2&amp;amp;px=400" role="button" title="guilherme_sales_0-1744302150241.png" alt="guilherme_sales_0-1744302150241.png" /&gt;&lt;/span&gt;&lt;/P&gt;&lt;P&gt;This is the code I am currently working in&amp;nbsp;&lt;A href="https://github.tools.sap/I538205/simple-saas-mt/tree/bwz" target="_blank" rel="nofollow noopener noreferrer"&gt;https://github.tools.sap/I538205/simple-saas-mt/tree/bwz&lt;/A&gt;&amp;nbsp;&lt;BR /&gt;and I also tested the sample from&amp;nbsp;&lt;A href="https://github.com/SAP-samples/build-workzone-integration/tree/main/standard/html5-content-provider-sample" target="_blank" rel="nofollow noopener noreferrer"&gt;https://github.com/SAP-samples/build-workzone-integration/tree/main/standard/html5-content-provider-sample&lt;/A&gt;&amp;nbsp;which also failed to import the content provider.&lt;/P&gt;&lt;P&gt;Has anyone else here faced this issue? How can I solve it?&lt;/P&gt;</content>
    <link href="https://community.sap.com/t5/technology-q-a/error-creating-content-provider-for-sap-build-work-zone-in-consumer/qaq-p/14072765"/>
    <published>2025-04-10T18:24:21.914000+02:00</published>
  </entry>
  <entry>
    <id>https://community.sap.com/t5/technology-q-a/unable-to-create-connection-from-sf-to-sac-amp-datasphere/qaq-p/14087434</id>
    <title>unable to create connection from SF to SAC &amp; datasphere</title>
    <updated>2025-04-27T08:45:21.188000+02:00</updated>
    <author>
      <name>DarrenEhsanSabzikari</name>
      <uri>https://community.sap.com/t5/user/viewprofilepage/user-id/1972419</uri>
    </author>
    <content>&lt;P&gt;Hi Experts,&amp;nbsp;&lt;/P&gt;&lt;P&gt;Please help me with this as im getting very annoyed on why we cannot create one simple connection, my intention is to create a connection between SF and SAC and Datasphere. could someone tell me the step-by-step process. I have searched many articular, but none has been very useful. i have screenshot from SF side with Oauth2 setup and SAC. the error i get is :&amp;nbsp;&lt;SPAN&gt;Unable to connect to datasource. Check the connection details and user credentials, and then try again.&lt;/SPAN&gt;&lt;/P&gt;&lt;P&gt;the question does this connection looks right and the USER id does it need special step that i have missed?&lt;/P&gt;&lt;P&gt;&lt;span class="lia-inline-image-display-wrapper lia-image-align-inline" image-alt="SF.png" style="width: 999px;"&gt;&lt;img src="https://community.sap.com/t5/image/serverpage/image-id/255218iE7728B1FADEBE929/image-size/large?v=v2&amp;amp;px=999" role="button" title="SF.png" alt="SF.png" /&gt;&lt;/span&gt;&lt;span class="lia-inline-image-display-wrapper lia-image-align-inline" image-alt="SAC.png" style="width: 955px;"&gt;&lt;img src="https://community.sap.com/t5/image/serverpage/image-id/255219iAA5300389CA68C17/image-size/large?v=v2&amp;amp;px=999" role="button" title="SAC.png" alt="SAC.png" /&gt;&lt;/span&gt;&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;</content>
    <link href="https://community.sap.com/t5/technology-q-a/unable-to-create-connection-from-sf-to-sac-amp-datasphere/qaq-p/14087434"/>
    <published>2025-04-27T08:45:21.188000+02:00</published>
  </entry>
  <entry>
    <id>https://community.sap.com/t5/technology-q-a/cloud-connector-access-to-al11-directory/qaq-p/14106715</id>
    <title>Cloud Connector access to AL11 directory</title>
    <updated>2025-05-20T14:48:40.954000+02:00</updated>
    <author>
      <name>Shubhash</name>
      <uri>https://community.sap.com/t5/user/viewprofilepage/user-id/2041566</uri>
    </author>
    <content>&lt;P&gt;Hi Experts,&lt;/P&gt;&lt;P&gt;I need your help to understand how to get the internal host of SFTP?&lt;/P&gt;&lt;P&gt;&lt;STRONG&gt;--Symptom--&lt;/STRONG&gt;&lt;/P&gt;&lt;UL&gt;&lt;LI&gt;We created few folders in AL11 directory, now we need to fetch the data from that folder to CPI via cloud connector to S4HANA Private Cloud.&amp;nbsp;&lt;/LI&gt;&lt;LI&gt;The blog is followed:&lt;BR /&gt;&lt;A href="https://community.sap.com/t5/technology-blog-posts-by-members/step-by-step-procedure-to-establish-cpi-to-sftp-server-connection/ba-p/13520842" target="_blank"&gt;https://community.sap.com/t5/technology-blog-posts-by-members/step-by-step-procedure-to-establish-cpi-to-sftp-server-connection/ba-p/13520842&lt;/A&gt;&lt;/LI&gt;&lt;/UL&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;&lt;P&gt;For Cloud Integration to access file storage, you must expose the file storage using FTP or SFTP protocol, by setting up FTP or SFTP service (as Server).&lt;/P&gt;&lt;P&gt;But how to get the internal host of SFTP?&lt;/P&gt;&lt;P&gt;Thank you very much.&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;</content>
    <link href="https://community.sap.com/t5/technology-q-a/cloud-connector-access-to-al11-directory/qaq-p/14106715"/>
    <published>2025-05-20T14:48:40.954000+02:00</published>
  </entry>
  <entry>
    <id>https://community.sap.com/t5/technology-q-a/sap-cap-502-quot-error-during-request-to-remote-service-read-econnreset/qaq-p/14142585</id>
    <title>SAP CAP : 502 "Error during request to remote service: read ECONNRESET"</title>
    <updated>2025-07-02T13:05:29.723000+02:00</updated>
    <author>
      <name>AakashN24</name>
      <uri>https://community.sap.com/t5/user/viewprofilepage/user-id/44956</uri>
    </author>
    <content>&lt;P&gt;Hi,&lt;/P&gt;&lt;P&gt;I am developing SAP CAP application, node.js, trying to connect to external S4 onPrem service which is accessed via the cloud connector.&lt;/P&gt;&lt;P&gt;I am trying to reach S4 OData service from CAP Application locally in Business Application Studio via SAP Cloud Connector.&lt;/P&gt;&lt;P&gt;To connect to the destination and other services we are using the ‚Äúcds bind -2‚Äù command, which created the .cdsrc-private.json file.&lt;/P&gt;&lt;P&gt;&lt;span class="lia-inline-image-display-wrapper lia-image-align-inline" image-alt=".cdsrc.jpg" style="width: 485px;"&gt;&lt;img src="https://community.sap.com/t5/image/serverpage/image-id/281736iD2235BDFAA2FFD25/image-dimensions/485x387?v=v2" width="485" height="387" role="button" title=".cdsrc.jpg" alt=".cdsrc.jpg" /&gt;&lt;/span&gt;&lt;/P&gt;&lt;P&gt;I am able to connect to the different BTP instances like the destination, authorization and connectivity instances, when i run the app using&amp;nbsp;&lt;/P&gt;&lt;P&gt;cds watch --profile hybrid&lt;/P&gt;&lt;P&gt;&lt;span class="lia-inline-image-display-wrapper lia-image-align-inline" image-alt="console1.png" style="width: 999px;"&gt;&lt;img src="https://community.sap.com/t5/image/serverpage/image-id/281737i15464359EE47790A/image-size/large?v=v2&amp;amp;px=999" role="button" title="console1.png" alt="console1.png" /&gt;&lt;/span&gt;&lt;/P&gt;&lt;P&gt;But when i'm trying to access any of the entities within the S4 OData service facing the following error,&lt;/P&gt;&lt;P&gt;{"error":{"code":"502","message":"Error during request to remote service: read ECONNRESET"}}&lt;/P&gt;&lt;P&gt;&lt;span class="lia-inline-image-display-wrapper lia-image-align-inline" image-alt="AakashN24_0-1751453547753.png" style="width: 652px;"&gt;&lt;img src="https://community.sap.com/t5/image/serverpage/image-id/281739iDB22D9D715C8A500/image-dimensions/652x178?v=v2" width="652" height="178" role="button" title="AakashN24_0-1751453547753.png" alt="AakashN24_0-1751453547753.png" /&gt;&lt;/span&gt;&lt;/P&gt;&lt;P&gt;&lt;span class="lia-inline-image-display-wrapper lia-image-align-inline" image-alt="AakashN24_1-1751453599642.png" style="width: 698px;"&gt;&lt;img src="https://community.sap.com/t5/image/serverpage/image-id/281747i75E3D45474ECA560/image-dimensions/698x100?v=v2" width="698" height="100" role="button" title="AakashN24_1-1751453599642.png" alt="AakashN24_1-1751453599642.png" /&gt;&lt;/span&gt;&lt;/P&gt;&lt;P&gt;&lt;SPAN&gt;Has anyone faced this issue, please suggest an appropriate solution for this.&lt;/SPAN&gt;&lt;/P&gt;&lt;P&gt;&lt;SPAN&gt;Also when the applicaiton is deployed to BTP,&amp;nbsp; the S4 odata services are accessiable via destination and the application is working as expected,&lt;/SPAN&gt;&lt;/P&gt;&lt;P&gt;&lt;SPAN&gt;i want run the application locally for development phase, hence need your help,&lt;/SPAN&gt;&lt;/P&gt;&lt;P&gt;&lt;SPAN&gt;Regards,&lt;/SPAN&gt;&lt;/P&gt;&lt;P&gt;&lt;SPAN&gt;Aakash&lt;/SPAN&gt;&lt;/P&gt;</content>
    <link href="https://community.sap.com/t5/technology-q-a/sap-cap-502-quot-error-during-request-to-remote-service-read-econnreset/qaq-p/14142585"/>
    <published>2025-07-02T13:05:29.723000+02:00</published>
  </entry>
  <entry>
    <id>https://community.sap.com/t5/technology-q-a/multiple-external-systems-connected-to-one-saprouter-in-sap-private-cloud/qaq-p/14142768</id>
    <title>Multiple external systems connected to one SAProuter in SAP Private Cloud</title>
    <updated>2025-07-02T16:12:39.277000+02:00</updated>
    <author>
      <name>andresalmansa</name>
      <uri>https://community.sap.com/t5/user/viewprofilepage/user-id/578295</uri>
    </author>
    <content>&lt;P&gt;We have SAP Private Cloud and we have added a SAProuter, and on the one hand we want to connect it to an external system for EDI, on the other hand we also have it for another system of connection with banks, but on the other hand we want to connect it to another external system of invoice management, but they tell us that it is not possible to connect to that SAProuter a third external system, honestly I thought that there were no limitations in connecting external systems to a single saprouter, I have been told that I would have to acquire a second saprouter, is this correct?&lt;/P&gt;&lt;P&gt;Regards&lt;/P&gt;&lt;P&gt;Andr√©s&lt;/P&gt;</content>
    <link href="https://community.sap.com/t5/technology-q-a/multiple-external-systems-connected-to-one-saprouter-in-sap-private-cloud/qaq-p/14142768"/>
    <published>2025-07-02T16:12:39.277000+02:00</published>
  </entry>
  <entry>
    <id>https://community.sap.com/t5/technology-q-a/oauth-technical-user-propagation-for-forms-service-by-adobe/qaq-p/14151340</id>
    <title>Oauth Technical User Propagation for Forms Service by Adobe</title>
    <updated>2025-07-14T06:40:48.422000+02:00</updated>
    <author>
      <name>Suneeth</name>
      <uri>https://community.sap.com/t5/user/viewprofilepage/user-id/1494905</uri>
    </author>
    <content>&lt;P&gt;Hello Experts,&lt;BR /&gt;We are trying to configure Oauth Technical User Propagation for usage by Forms Service by Adobe on SAP BTP by following the below guides&lt;/P&gt;&lt;P&gt;&lt;A href="https://help.sap.com/docs/forms-service-by-adobe/sap-forms-service-cf/configure-technical-user-propagation" target="_blank" rel="noopener noreferrer"&gt;https://help.sap.com/docs/forms-service-by-adobe/sap-forms-service-cf/configure-technical-user-propagation&lt;/A&gt;&lt;/P&gt;&lt;P&gt;The backend system is a S4 HANA 2023 system . ADS works fine when we use basic authentication.&lt;/P&gt;&lt;P&gt;The BTP subaccounts, cloud connector and backend S4 HANA 2023 system are all set for Principal Propagation and normal calls via Principal Propagation works fine&lt;BR /&gt;As per the document for Forms Service, to use oauth technical user propagation - we have to create a destination using the proxy type 'on-premise' and so expected to go via the cloud connector&lt;BR /&gt;&lt;BR /&gt;However, SAP documentation &lt;A href="https://help.sap.com/docs/connectivity/sap-btp-connectivity-cf/oauth-technical-user-propagation-authentication" target="_self" rel="noopener noreferrer"&gt;here&lt;/A&gt; mentions that the calls to token Service URL does not go via the cloud connector. However, it still does not allow us to enter a HTTPS URL for the 'token service URL '&amp;nbsp; (any service credentials created in the BTP subaccount has a https URL for token service ) . Anyhow - we tried replacing HTTPS with HTTP for the token URL (and also creating a destination in cloud connector to redirect it to HTTPS ) but it does not help . There are no calls logged in the cloud connector as well&lt;/P&gt;&lt;P&gt;The token URl and credentials we used was from a "Authorization and Trust MAnagement Service" in the same subaccount as described as an example in the Forms Service guide&lt;/P&gt;&lt;P&gt;&lt;span class="lia-inline-image-display-wrapper lia-image-align-inline" image-alt="Suneeth_0-1752465324221.png" style="width: 400px;"&gt;&lt;img src="https://community.sap.com/t5/image/serverpage/image-id/285982i944172F51D862338/image-size/medium?v=v2&amp;amp;px=400" role="button" title="Suneeth_0-1752465324221.png" alt="Suneeth_0-1752465324221.png" /&gt;&lt;/span&gt;&lt;/P&gt;&lt;P&gt;How does this work ?&lt;/P&gt;&lt;P&gt;&lt;BR /&gt;&lt;BR /&gt;&lt;/P&gt;</content>
    <link href="https://community.sap.com/t5/technology-q-a/oauth-technical-user-propagation-for-forms-service-by-adobe/qaq-p/14151340"/>
    <published>2025-07-14T06:40:48.422000+02:00</published>
  </entry>
  <entry>
    <id>https://community.sap.com/t5/technology-q-a/abap-platform-pipeline-in-sap-ci-cd-service-release-issue/qaq-p/14166006</id>
    <title>ABAP Platform pipeline in SAP CI/CD Service - release issue</title>
    <updated>2025-07-29T17:23:05.180000+02:00</updated>
    <author>
      <name>AdrDen</name>
      <uri>https://community.sap.com/t5/user/viewprofilepage/user-id/362972</uri>
    </author>
    <content>&lt;P&gt;Hi,&amp;nbsp;&lt;/P&gt;&lt;P&gt;I have a problem with my &lt;U&gt;ABAP Platform pipeline&lt;/U&gt;, using SAP Continuous Integration and Delivery Service in BTP. The issue is with the&amp;nbsp;&lt;STRONG&gt;Release&lt;/STRONG&gt; part.&lt;/P&gt;&lt;P&gt;I have a Cloud Connector and destination (using Principal Propagation) ready. The deployment from SAP BAS works as expected. All necessary OData Services are active and whitelisted. However, there's 401 error when the pipeline is executed.&lt;/P&gt;&lt;PRE&gt;[2025-07-29T14:59:03.441Z] Sending GET request to http://XXX:400/sap/opu/odata/UI5/ABAP_REPOSITORY_SRV/
[2025-07-29T14:59:03.677Z] Release stage credentials successfully validated against endpoint&lt;BR /&gt;...&lt;/PRE&gt;&lt;PRE&gt;[2025-07-29T14:59:23.345Z] info  transportRequestUploadCTS - No deploy dependencies provided. Skipping npm install call. Assuming current docker image already contains the dependencies for performing the deployment.
[2025-07-29T14:59:23.345Z] info  transportRequestUploadCTS - running shell script: /bin/sh #!/bin/sh -e
[2025-07-29T14:59:23.345Z] fiori deploy --failfast --yes --username ABAP_USER --password ABAP_PASSWORD --description "Pipeline Test" --config "ui5-deploy.yaml" --url http://XXX:400 --transport GXXXXXXXX --package Z_PACKAGE --name Z_TEST
[2025-07-29T14:59:24.278Z] info  transportRequestUploadCTS - &amp;#27;[32minfo&amp;#27;[39m abap-deploy-task Z_TEST Creating archive with UI5 build result.
[2025-07-29T14:59:24.278Z] info  transportRequestUploadCTS - &amp;#27;[32minfo&amp;#27;[39m abap-deploy-task Z_TEST Archive created.
[2025-07-29T14:59:24.278Z] info  transportRequestUploadCTS - &amp;#27;[32minfo&amp;#27;[39m abap-deploy-task Z_TEST Starting to deploy.
[2025-07-29T14:59:24.278Z] info  transportRequestUploadCTS - &amp;#27;[31merror&amp;#27;[39m abap-deploy-task Z_TEST Deployment has failed.
[2025-07-29T14:59:24.278Z] info  transportRequestUploadCTS - &amp;#27;[31merror&amp;#27;[39m abap-deploy-task Z_TEST Change logging level to debug your issue
[2025-07-29T14:59:24.278Z] info  transportRequestUploadCTS - &amp;#27;[31merror&amp;#27;[39m abap-deploy-task Z_TEST 	(see examples https://github.com/SAP/open-ux-tools/tree/main/packages/deploy-tooling#configuration-with-logging-enabled)
[2025-07-29T14:59:24.278Z] info  transportRequestUploadCTS - &amp;#27;[31merror&amp;#27;[39m builder:&lt;STRONG&gt;custom deploy-to-abap Request failed with status code 401&lt;/STRONG&gt;
[2025-07-29T14:59:24.279Z] info  transportRequestUploadCTS - Command deploy failed with error : &lt;STRONG&gt;Request failed with status code 401&lt;/STRONG&gt;
[2025-07-29T14:59:42.331Z] info  transportRequestUploadCTS - fatal error: errorDetails{"category":"undefined","correlationId":"n/a","error":"&lt;STRONG&gt;running shell script failed with /bin/sh: cmd.Run() failed: exit status 1&lt;/STRONG&gt;","library":"SAP/jenkins-library","message":"step execution failed","result":"failure","stepName":"transportRequestUploadCTS","time":"2025-07-29T14:59:39.890292594Z"}
[2025-07-29T14:59:42.332Z] fatal transportRequestUploadCTS - step execution failed - running shell script failed with /bin/sh: cmd.Run() failed: exit status 1&lt;/PRE&gt;&lt;P&gt;&amp;nbsp;I use &lt;STRONG&gt;Job Editor&lt;/STRONG&gt; configuration mode. No code is used for the pipeline.&amp;nbsp;&lt;/P&gt;&lt;P&gt;Upload credentials is Basic Auth. using my Username and Password. These are no issues with it, as I've tested successfully.&lt;/P&gt;&lt;P&gt;I also use Cloud Connector, with Location ID specified on the destination level.&lt;/P&gt;&lt;P&gt;&lt;span class="lia-inline-image-display-wrapper lia-image-align-inline" image-alt="AdrDen_0-1753802156650.png" style="width: 400px;"&gt;&lt;img src="https://community.sap.com/t5/image/serverpage/image-id/293514i9ADDCA2A04833BB2/image-size/medium?v=v2&amp;amp;px=400" role="button" title="AdrDen_0-1753802156650.png" alt="AdrDen_0-1753802156650.png" /&gt;&lt;/span&gt;&lt;/P&gt;&lt;P&gt;I have a couple of doubts:&lt;/P&gt;&lt;P&gt;- I see the&amp;nbsp;--config "ui5-deploy.yaml" is used. Is there any special ui5-deploy content I have to add? I use destination there.&lt;/P&gt;&lt;P&gt;- Is ABAP Platform Endpoint the URL that I can find on my Destination level?&lt;/P&gt;&lt;P&gt;- Location ID - documentation says it should be found on the Cloud Connector level in BTP Cockpit, but it's false. There's Location ID, but only on the Destination level.&lt;/P&gt;&lt;P&gt;Any idea what could be the issue?&lt;/P&gt;&lt;P&gt;BR,&lt;/P&gt;&lt;P&gt;Adrian&lt;/P&gt;&lt;P&gt;&amp;nbsp;&lt;/P&gt;</content>
    <link href="https://community.sap.com/t5/technology-q-a/abap-platform-pipeline-in-sap-ci-cd-service-release-issue/qaq-p/14166006"/>
    <published>2025-07-29T17:23:05.180000+02:00</published>
  </entry>
  <entry>
    <id>https://community.sap.com/t5/technology-q-a/ads-configuration-via-oauth2technicaluserpropagation/qaq-p/14185943</id>
    <title>ADS configuration via OAuth2TechnicalUserPropagation</title>
    <updated>2025-08-21T08:14:19.320000+02:00</updated>
    <author>
      <name>bsamit15</name>
      <uri>https://community.sap.com/t5/user/viewprofilepage/user-id/1999299</uri>
    </author>
    <content>&lt;P&gt;Hi,&lt;/P&gt;&lt;P&gt;Hi , I am having trouble in ADS configuration via&amp;nbsp;OAuth2TechnicalUserPropagation. Reports is throwing an below error.&lt;/P&gt;&lt;P&gt;We have already set the cloud connector setting to X.509.&lt;/P&gt;&lt;H3 id="toc-hId-1866983916"&gt;com.adobe.ads.destination.DestinationException: HTTP-401 Error.(200.101).&lt;/H3&gt;&lt;P&gt;I have done the principal propagation as per the SNOTE 345281 and its working fine. We have tested for BAS application.&lt;/P&gt;&lt;P&gt;With Basic authentication reports are working fine.&lt;/P&gt;</content>
    <link href="https://community.sap.com/t5/technology-q-a/ads-configuration-via-oauth2technicaluserpropagation/qaq-p/14185943"/>
    <published>2025-08-21T08:14:19.320000+02:00</published>
  </entry>
  <entry>
    <id>https://community.sap.com/t5/technology-q-a/bind-connectivity-service-to-bas-doesn-t-work-and-end-up-in-cf/qaq-p/14229565</id>
    <title>bind connectivity service to bas doesn't work and end up in  CF-UnprocessableEntity error</title>
    <updated>2025-09-27T16:15:07.293000+02:00</updated>
    <author>
      <name>haschneider</name>
      <uri>https://community.sap.com/t5/user/viewprofilepage/user-id/878739</uri>
    </author>
    <content>&lt;P&gt;Hello,&lt;/P&gt;&lt;P&gt;i have the problem that i can't bind the connectivity service in my Business Application Studio. If i try to bind the service over the cloudfoundry targets menu then i get the following error.&amp;nbsp;&lt;/P&gt;&lt;P&gt;&lt;span class="lia-inline-image-display-wrapper lia-image-align-inline" image-alt="haschneider_0-1758982421535.png" style="width: 400px;"&gt;&lt;img src="https://community.sap.com/t5/image/serverpage/image-id/320720i16643F4815F1279F/image-size/medium?v=v2&amp;amp;px=400" role="button" title="haschneider_0-1758982421535.png" alt="haschneider_0-1758982421535.png" /&gt;&lt;/span&gt;&lt;/P&gt;&lt;P&gt;CF-UnprocessableEntity (Assign a droplet before starting this app.) [10008]&lt;/P&gt;&lt;P&gt;Has anyone an idea why it is so an how i can resolve this problem?&amp;nbsp;&lt;/P&gt;</content>
    <link href="https://community.sap.com/t5/technology-q-a/bind-connectivity-service-to-bas-doesn-t-work-and-end-up-in-cf/qaq-p/14229565"/>
    <published>2025-09-27T16:15:07.293000+02:00</published>
  </entry>
</feed>
